src/test/java/io/confluent/connect/jdbc/sink/JdbcSinkConfigTest.java

```diff
@@ -86,6 +86,13 @@ public void shouldCreateConfigWithTableOnly() {
     assertTableTypes(TableType.TABLE);
   }
 
+  @Test
+  public void shouldCreateConfigWithPartitionedTableOnly() {
+    props.put("table.types", "partitioned table");
```

Comment:
Not sure if the space in the value here might interfere with configuration; review welcome!

Reply:
Would you not want this to line up with [the change in the driver](https://github.com/pgjdbc/pgjdbc/commit/25eb32c8681eaa4aaac801808b6028e9f5dfbea8#diff-0571f8ac3385a7f7bb34e5c77f8afd24810311506989379c2e85c6c16eea6ce4R1287)?

Reply:
I don't think space would cause any issues. I checked for JDBC source, table.types config also takes "System table" as an input. So I think it should be fine here as well.